src/features/contentCreator/createSampleExecutionEnvPage.ts

```diff
@@ -312,7 +314,7 @@ export class CreateSampleExecutionEnv {
       commandOutput += ansibleCreatorExecutionResult.output;
       commandResult = ansibleCreatorExecutionResult.status;
     } else {
-      commandOutput += `Minimum ansible-creator version needed to scaffold an execution-environment.yml file is 25.1.0\n`;
+      commandOutput += `Minimum ansible-creator version needed to scaffold an execution-environment.yml file is 24.12.1\n`;
```

Comment:
@abhikdps we should use the `ANSIBLE_CREATOR_EE_VERSION_MIN` variable instead of hardcoding this value here.